docs/user/troubleshooting-guides/03-70-empty-apirule-spec.md

```diff
@@ -0,0 +1,27 @@
+# APIRule Contains an Empty Spec
+
+## Symptoms
+
+- There is an empty spec in an `APIRule`, for example:
+  ```yaml
+  apiVersion: gateway.kyma-project.io/v2alpha1
+  kind: APIRule
+  metadata:
+    name: httpbin
+    namespace: test
+  spec: {}
+  status:
+    lastProcessedTime: "2024-07-22T07:06:59Z"
+    state: Ready
+  ```
+
+## Cause
+
+The conversion between the `APIRule` CRD versions is not possible. It's only possible to convert `jwt` handler from `v1beta1` to `v2alpha1`, for more information see the [documentation](https://github.com/kyma-project/api-gateway/blob/main/docs/user/custom-resources/apirule/v2alpha1/04-60-apirule-migration.md).
```

Comment:
That's not 100% correct, `no_auth` can also be converted from v1beta1 to v2alpha1. But jwt handler with more than one `trusted_issuers` or `jwks_urls` can also not be converted.

The linked migration documentation doesn't help much in this context. It would be helpful to link the migration guide, but it doesn't exist, yet.